substrate/client/rpc-spec-v2/Cargo.toml

```
@@ -44,6 +44,7 @@ futures-util = { version = "0.3.30", default-features = false }
 rand = "0.8.5"
 
 [dev-dependencies]
+jsonrpsee = { version = "0.22", features = ["client-core", "macros", "client", "server"] }
```

"client", will add lots of dependencies AFAIU "ws-client" and "server" should be sufficient

```suggestion
jsonrpsee = { version = "0.22", features = ["ws-client", "server"] }
```